### ISSUE TITLE
Provide link to the source for the content in open source section

### ISSUE THREAD

**Issue by palash25** [dipakkr/A-to-Z-Resources-for-Students#1064]
Hi @dipakkr you have put together an awesome list :clap: 

I have been contributing to this list for a long time https://github.com/tapasweni-pathak/SOC-Programs so I couldn't help but notice the similarity between your list's open source section and the contents of the SOC programs list.

Since the list is open source there is nothing wrong with copying the contents of it but I think it would be nice if you could point the visitors of your list to the original one in case the contents of this one ever go out of sync with the original list and also to give credit to all the different contributors who helped build that list. :smile:

**Comment by dipakkr** [comment]
Hi @palash25 thanks for pointing out. I found that this list was contributed by someone ( contributor ). I have received a lot of PR, and have also rejected many because they were copied from someone else repository. However, I failed to identify this section. Earlier I had made a different list for open source section, but I found this list good and merged this, but didn't know it was copied. 

I will do the needful asap. 

Thanks !!
